test/e2e/dashboards.go

```diff
@@ -21,6 +21,12 @@ var (
 		"grafana-dashboard-definition-knative-eventing-channel",
 		"grafana-dashboard-definition-knative-eventing-kafka-sink",
 	}
+
+	ServingDashboards = []string{
```

Comment:
Is this a separate thing or related to the description/rest in this PR?

Reply:
Separate but saw the opportunity to just complete the tests.